diff --git a/node_modules/react-native-audio-api/android/build.gradle b/node_modules/react-native-audio-api/android/build.gradle
index 49b6f51..be4940e 100644
--- a/node_modules/react-native-audio-api/android/build.gradle
+++ b/node_modules/react-native-audio-api/android/build.gradle
@@ -306,10 +306,10 @@ def assertMinimalReactNativeVersion = task assertMinimalReactNativeVersionTask {
     }
 }
 
-task downloadPrebuiltBinaries(type: Exec) {
-  commandLine 'chmod', '+x', '../scripts/download-prebuilt-binaries.sh'
-  commandLine 'bash', '../scripts/download-prebuilt-binaries.sh'
-  args 'android', isFFmpegDisabled() ? 'skipffmpeg' : ''
+task downloadPrebuiltBinaries {
+  doLast {
+      println "Skipping downloadPrebuiltBinaries (handled by custom script)"
+  }
 }
 
 // Make preBuild depend on the download task
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/._jniLibs b/node_modules/react-native-audio-api/android/src/main/__MACOSX/._jniLibs
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/._jniLibs differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/._arm64-v8a b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/._arm64-v8a
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/._arm64-v8a differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/._armeabi-v7a b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/._armeabi-v7a
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/._armeabi-v7a differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/._x86 b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/._x86
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/._x86 differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/._x86_64 b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/._x86_64
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/._x86_64 differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/arm64-v8a/._libavcodec.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/arm64-v8a/._libavcodec.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/arm64-v8a/._libavcodec.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/arm64-v8a/._libavformat.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/arm64-v8a/._libavformat.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/arm64-v8a/._libavformat.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/arm64-v8a/._libavutil.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/arm64-v8a/._libavutil.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/arm64-v8a/._libavutil.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/arm64-v8a/._libswresample.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/arm64-v8a/._libswresample.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/arm64-v8a/._libswresample.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/armeabi-v7a/._libavcodec.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/armeabi-v7a/._libavcodec.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/armeabi-v7a/._libavcodec.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/armeabi-v7a/._libavformat.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/armeabi-v7a/._libavformat.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/armeabi-v7a/._libavformat.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/armeabi-v7a/._libavutil.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/armeabi-v7a/._libavutil.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/armeabi-v7a/._libavutil.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/armeabi-v7a/._libswresample.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/armeabi-v7a/._libswresample.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/armeabi-v7a/._libswresample.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86/._libavcodec.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86/._libavcodec.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86/._libavcodec.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86/._libavformat.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86/._libavformat.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86/._libavformat.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86/._libavutil.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86/._libavutil.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86/._libavutil.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86/._libswresample.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86/._libswresample.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86/._libswresample.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86_64/._libavcodec.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86_64/._libavcodec.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86_64/._libavcodec.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86_64/._libavformat.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86_64/._libavformat.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86_64/._libavformat.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86_64/._libavutil.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86_64/._libavutil.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86_64/._libavutil.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86_64/._libswresample.so b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86_64/._libswresample.so
new file mode 100644
index 0000000..06c6eed
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/__MACOSX/jniLibs/x86_64/._libswresample.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/arm64-v8a/libavcodec.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/arm64-v8a/libavcodec.so
new file mode 100644
index 0000000..96442b8
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/arm64-v8a/libavcodec.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/arm64-v8a/libavformat.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/arm64-v8a/libavformat.so
new file mode 100644
index 0000000..60ac543
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/arm64-v8a/libavformat.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/arm64-v8a/libavutil.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/arm64-v8a/libavutil.so
new file mode 100644
index 0000000..c9060e7
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/arm64-v8a/libavutil.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/arm64-v8a/libswresample.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/arm64-v8a/libswresample.so
new file mode 100644
index 0000000..1c713b2
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/arm64-v8a/libswresample.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/armeabi-v7a/libavcodec.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/armeabi-v7a/libavcodec.so
new file mode 100644
index 0000000..77eac12
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/armeabi-v7a/libavcodec.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/armeabi-v7a/libavformat.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/armeabi-v7a/libavformat.so
new file mode 100644
index 0000000..4586d12
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/armeabi-v7a/libavformat.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/armeabi-v7a/libavutil.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/armeabi-v7a/libavutil.so
new file mode 100644
index 0000000..e2254b4
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/armeabi-v7a/libavutil.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/armeabi-v7a/libswresample.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/armeabi-v7a/libswresample.so
new file mode 100644
index 0000000..c3e40d2
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/armeabi-v7a/libswresample.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/x86/libavcodec.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86/libavcodec.so
new file mode 100644
index 0000000..6133b04
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86/libavcodec.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/x86/libavformat.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86/libavformat.so
new file mode 100644
index 0000000..251e8a8
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86/libavformat.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/x86/libavutil.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86/libavutil.so
new file mode 100644
index 0000000..6c2811b
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86/libavutil.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/x86/libswresample.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86/libswresample.so
new file mode 100644
index 0000000..4447ff6
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86/libswresample.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/x86_64/libavcodec.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86_64/libavcodec.so
new file mode 100644
index 0000000..a5d36ac
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86_64/libavcodec.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/x86_64/libavformat.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86_64/libavformat.so
new file mode 100644
index 0000000..61391b9
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86_64/libavformat.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/x86_64/libavutil.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86_64/libavutil.so
new file mode 100644
index 0000000..115f47a
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86_64/libavutil.so differ
diff --git a/node_modules/react-native-audio-api/android/src/main/jniLibs/x86_64/libswresample.so b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86_64/libswresample.so
new file mode 100644
index 0000000..225e5e4
Binary files /dev/null and b/node_modules/react-native-audio-api/android/src/main/jniLibs/x86_64/libswresample.so differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libcrypto.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libcrypto.a
new file mode 100644
index 0000000..5cd516d
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libcrypto.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libogg.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libogg.a
new file mode 100644
index 0000000..67c00a4
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libogg.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libopus.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libopus.a
new file mode 100644
index 0000000..f1f6b24
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libopus.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libopusfile.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libopusfile.a
new file mode 100644
index 0000000..71a9ae4
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libopusfile.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libssl.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libssl.a
new file mode 100644
index 0000000..6a90555
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libssl.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libvorbis.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libvorbis.a
new file mode 100644
index 0000000..8f50c53
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libvorbis.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libvorbisenc.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libvorbisenc.a
new file mode 100644
index 0000000..f27d5b2
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libvorbisenc.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libvorbisfile.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libvorbisfile.a
new file mode 100644
index 0000000..371e758
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/arm64-v8a/libvorbisfile.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libcrypto.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libcrypto.a
new file mode 100644
index 0000000..fbf39a5
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libcrypto.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libogg.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libogg.a
new file mode 100644
index 0000000..537c72d
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libogg.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libopus.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libopus.a
new file mode 100644
index 0000000..a6f1533
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libopus.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libopusfile.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libopusfile.a
new file mode 100644
index 0000000..6a7a5e9
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libopusfile.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libssl.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libssl.a
new file mode 100644
index 0000000..226749d
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libssl.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libvorbis.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libvorbis.a
new file mode 100644
index 0000000..c34bcc7
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libvorbis.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libvorbisenc.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libvorbisenc.a
new file mode 100644
index 0000000..f65f760
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libvorbisenc.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libvorbisfile.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libvorbisfile.a
new file mode 100644
index 0000000..1c7a04e
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/armeabi-v7a/libvorbisfile.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libcrypto.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libcrypto.a
new file mode 100644
index 0000000..16d29f7
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libcrypto.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libogg.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libogg.a
new file mode 100644
index 0000000..f0deee4
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libogg.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libopus.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libopus.a
new file mode 100644
index 0000000..565c451
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libopus.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libopusfile.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libopusfile.a
new file mode 100644
index 0000000..5cbd1ad
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libopusfile.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libssl.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libssl.a
new file mode 100644
index 0000000..0a5f27d
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libssl.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libvorbis.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libvorbis.a
new file mode 100644
index 0000000..e0ead0c
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libvorbis.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libvorbisenc.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libvorbisenc.a
new file mode 100644
index 0000000..e799d77
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libvorbisenc.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libvorbisfile.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libvorbisfile.a
new file mode 100644
index 0000000..1dfc661
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86/libvorbisfile.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libcrypto.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libcrypto.a
new file mode 100644
index 0000000..be88da9
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libcrypto.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libogg.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libogg.a
new file mode 100644
index 0000000..509d35c
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libogg.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libopus.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libopus.a
new file mode 100644
index 0000000..7eb0d45
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libopus.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libopusfile.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libopusfile.a
new file mode 100644
index 0000000..77711f4
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libopusfile.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libssl.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libssl.a
new file mode 100644
index 0000000..9091b2e
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libssl.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libvorbis.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libvorbis.a
new file mode 100644
index 0000000..cb966fe
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libvorbis.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libvorbisenc.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libvorbisenc.a
new file mode 100644
index 0000000..f0aa17f
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libvorbisenc.a differ
diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libvorbisfile.a b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libvorbisfile.a
new file mode 100644
index 0000000..dfb7753
Binary files /dev/null and b/node_modules/react-native-audio-api/common/cpp/audioapi/external/x86_64/libvorbisfile.a differ
